src/tests/notices/getNoticeDetails.js

```diff
@@ -0,0 +1,52 @@
+import { assert, statusOk } from "../../common/assertions.js";
+import {
+  noticeDetails,
+  NOTICES_API_NAMES,
+} from "../../api/notices.js";
+import defaultHandleSummaryBuilder from "../../common/handleSummaryBuilder.js";
+import { defaultApiOptionsBuilder } from "../../common/dynamicScenarios/defaultOptions.js";
+import { logErrorResult } from "../../common/dynamicScenarios/utils.js";
+import { getAuthToken,abort,getTestEntity } from "../../common/utils.js";
+import { getNoticesList } from "./getNoticesList.js"
+
+const application = "notices";
+const testName = "getNoticeDetails";
+const ERROR_MESSAGE = "No elements found in notice list please restart test with at least one element";
+
+// Dynamic scenarios' K6 configuration
+export const options = defaultApiOptionsBuilder(
+  application,
+  testName,
+  [
+    NOTICES_API_NAMES.getNoticeDetails,
+  ] // applying apiName tags to thresholds
+);
+
+// K6 summary configuration
+export const handleSummary = defaultHandleSummaryBuilder(application, testName);
+
+export function setup() {
+  const authToken = getAuthToken();
+  const noticesList = getNoticesList(authToken).json().notices;
+  return {
+    token: authToken,
+    notices: noticesList.length === 0 ? abort(ERROR_MESSAGE) : noticesList.map(item => item.eventId),
```

Comment:
don't use a ternary operator! use an if statement!
the ternary is better to be used if you have to effectively give another value to the variable... in this case you are aborting the execution... not assigning another value